debezium-connector-mongodb/src/main/java/io/debezium/connector/mongodb/sink/MongoDbSinkConnectorConfig.java

```diff
@@ -0,0 +1,232 @@
+            .withDescription("The name of the MongoDB database to which the connector writes to.")
+            .required();
+
+    public static final Field TABLE_NAME_FORMAT_FIELD = Field.create(TABLE_NAME_FORMAT)
+            .withDisplayName("A format string for the table")
+            .withType(ConfigDef.Type.STRING)
+            .withGroup(Field.createGroupEntry(Field.Group.CONNECTOR, 3))
+            .withWidth(ConfigDef.Width.MEDIUM)
+            .withImportance(ConfigDef.Importance.MEDIUM)
+            .withDefault("${topic}")
+            .withDescription("A format string for the table, which may contain '${topic}' as a placeholder for the original topic name.");
+
+    public static final Field TABLE_NAMING_STRATEGY_FIELD = Field.create(TABLE_NAMING_STRATEGY)
+            .withDisplayName("Name of the strategy class that implements the TablingNamingStrategy interface")
+            .withType(ConfigDef.Type.CLASS)
+            .withGroup(Field.createGroupEntry(Field.Group.CONNECTOR_ADVANCED, 2))
+            .withWidth(ConfigDef.Width.LONG)
+            .withImportance(ConfigDef.Importance.LOW)
+            .withDefault(DefaultTableNamingStrategy.class.getName())
+            .withDescription("Name of the strategy class that implements the TableNamingStrategy interface.");
+
+    public static final Field COLUMN_NAMING_STRATEGY_FIELD = Field.create(COLUMN_NAMING_STRATEGY)
+            .withDisplayName("Name of the strategy class that implements the ColumnNamingStrategy interface")
+            .withType(ConfigDef.Type.CLASS)
+            .withGroup(Field.createGroupEntry(Field.Group.CONNECTOR_ADVANCED, 3))
+            .withWidth(ConfigDef.Width.LONG)
+            .withImportance(ConfigDef.Importance.LOW)
+            .withDefault(DefaultColumnNamingStrategy.class.getName())
+            .withDescription("Name of the strategy class that implements the ColumnNamingStrategy interface.");
+
+    public static final Field BATCH_SIZE_FIELD = Field.create(BATCH_SIZE)
+            .withDisplayName("Specifies how many records to attempt to batch together into the destination table, when possible. " +
+                    "You can also configure the connector’s underlying consumer’s max.poll.records using consumer.override.max.poll.records in the connector configuration.")
+            .withType(ConfigDef.Type.INT)
+            .withGroup(Field.createGroupEntry(Field.Group.CONNECTOR_ADVANCED, 4))
+            .withWidth(ConfigDef.Width.SHORT)
+            .withImportance(ConfigDef.Importance.MEDIUM)
+            .withDefault(500)
+            .withDescription("Specifies how many records to attempt to batch together into the destination table, when possible. " +
+                    "You can also configure the connector’s underlying consumer’s max.poll.records using consumer.override.max.poll.records in the connector configuration.");
+
+    // public static final Field FIELD_INCLUDE_LIST_FIELD = Field.create(FIELD_INCLUDE_LIST)
```

Comment:
Lets remove these commented bits if they're not being used.

Reply:
same.

Reply:
done